internal/backend/svc/banner.txt

```diff
@@ -1,8 +1,8 @@
 
-        _..--~~~~--.._             autokitteh   {{ .Version }}
-     _--              --_          PID:         {{ .PID }}
-    '                    `         gRPC/HTTP:   {{ .Addr }}
-   '                      `        Web:         {{ .WebAddr }}
+        _..--~~~~--.._             autokitteh    {{ .Version }}
+     _--              --_          PID:          {{ .PID }}
+    '                    `         gRPC/HTTP:    {{ .Addr }}
+   '                      `        Web{{ .WebVer }}  {{ .WebAddr }}
```

Comment:
It's very easy to make this look messed up when you inject a variable-length string in the left side, especially with no space after "Web", and no ":" before the address. Maybe `Web: address (version)` instead? Or separate "Web Address" and "Web Version"?

Reply:
in a futute pr.